packages/eth2.0-utils/src/yaml/index.ts

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line import/default
+import camelcase from "camelcase";
+import {load} from "js-yaml";
+import {readFileSync} from "fs";
+import {schema} from "./schema";
+
+export * from "./expandYamlValue";
+
+export function objectToCamelCase(obj: object): object {
+  if (Object(obj) === obj) {
+    Object.getOwnPropertyNames(obj).forEach((name) => {
+      const newName = camelcase(name);
+      if (newName !== name) {
+        // @ts-ignore
+        obj[newName] = obj[name];
+        // @ts-ignore
+        delete obj[name];
+      }
+      // @ts-ignore
+      objectToCamelCase(obj[newName]);
+    });
+  } else if (Array.isArray(obj)) {
+    for (let i = 0; i < obj.length; i++) {
+      obj[i] = objectToCamelCase(obj[i]);
+    }
+  }
+  return obj;
+}
+
+export function loadYamlFile(path: string): object {
```

Comment:
Can we break out this function like
https://github.com/ChainSafe/lodestar/pull/432/files#diff-e634a0f8e377b9fa0942cdf4b029afe7R25

Reply:
sure

Reply:
I will remove that function as it will go in ssz